socorro/external/elasticsearch/index_cleaner.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import pyelasticsearch
+import re
+
+from configman import Namespace, RequiredConfig
+from configman.converters import class_converter
+
+from socorro.lib.datetimeutil import utc_now
+
+
+class IndexCleaner(RequiredConfig):
+    """Delete old Elasticsearch indices from our databases. """
+
+    required_config = Namespace()
+    required_config.add_option(
+        'retention_policy',
+        default=26,
+        doc='Number of weeks to keep an index alive. ',
+    )
+    required_config.namespace('elasticsearch')
+    required_config.elasticsearch.add_option(
+        'elasticsearch_class',
+        default='socorro.external.elasticsearch.connection_context.'
+                'ConnectionContext',
+        from_string_converter=class_converter,
+        reference_value_from='resource.elasticsearch',
+    )
+    required_config.elasticsearch.add_option(
+        'elasticsearch_index_regex',
+        default='socorro[0-9]{6}$',
```

Comment:
In the interest of safety, this regex could (should?) include the starts-with prefix as well, as in `^socorro[0-9]{6}$'` (note the caret).